src/card-list.js

```diff
@@ -0,0 +1,81 @@
+import { LitElement, html } from 'lit';
+import equalizer from '../application.js';
+import styles from './card-list-styles.js';
+
+class CardList extends LitElement {
+
+	static get properties() {
+		return {
+			books: [],
+			data: [],
+			icon: '',
+			placeholder: '',
+		}
+	}
+
+	static get styles() {
+		return [equalizer, styles];
+	}
+
+	constructor() {
+		super()
+		this.placeholder = "Buscá por título del libro...";
+		this.books = [];
+		this.getDataJSON();
+		this.icon = "./assets/images/Search-Button.png";
+	}
+
+	async getDataJSON() {
+		const data = await fetch('../json/books.json').then(response => response.json());
+		this.data = data;
+		this.ShortData();
+	}
+
+	ShortData() {
+		const temporalData = [];
+		this.data.map(element => {
+			temporalData.push({ img: element.img, book_title: element.book_title, author: element.author });
+		});
+		this.books = temporalData;
```

Comment:
the idea of using map isn't to have a `temporalData` variable, you can do this
```suggestion
		this.books = this.data.map(({img, book_title, author}) => 
		  ({ img, book_title, author })
		);
		
```